.ci/setup.sh

```diff
@@ -29,7 +29,7 @@ if [[ $OS_NAME == "macos" ]]; then
         brew install swig
     fi
 else  # Linux
-    if type -f apt 2>&1 > /dev/null; then
+    if { type -f apt > /dev/null; } 2>&1; then
```

Comment:
What is the specific `shellcheck` warning this is intended to address?

Reply:
The warning message is 

                   ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
see                    ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

Reply:
```suggestion
    if type -f apt > /dev/null 2>&1; then
```

Thanks. Please just move the `2>&1` to the end and don't introduce a `{` / `}`.